drivers/volume/portworx/portworx.go

```diff
@@ -1263,6 +1263,33 @@ func (d *portworx) ValidateRebalanceJobs() error {
 	return nil
 }
 
+func (d *portworx) ResizeStoragePoolByPercentage(poolUUID string, e api.SdkStoragePool_ResizeOperationType, percentage uint64) error {
+
+	// start a task to check if pool  resize is done
+	t := func() (interface{}, bool, error) {
+		logrus.Infof("Initiating pool %v resize by %v with operationtype %v", poolUUID, percentage, e.String())
```

Comment:
move this print message out of the retry loop, we don't want to see this message every time when it does retry.

Reply:
done